main.py

```diff
@@ -1,10 +1,58 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_races(players: dict) -> None:
```

Comment:
consider to change function names. You not just create races, but also skills. Same in the next func